test/test/kernel/KernelTests.java

```diff
@@ -0,0 +1,27 @@
+package test.kernel;
+
+import one.profiler.test.Output;
+import one.profiler.test.Test;
+import one.profiler.test.TestProcess;
+
+public class KernelTests {
```

Comment:
Does it run on macOS? Or where do you specify it is Linux-only test? (macOS does not support kernel profiling)
The same applies to hw counters tests.

`make test` should pass both on Linux and macOS. If it doesn't, it's ok to disable some platform-specific tests for now; you may enable them in the separate PR.